client/pages/home.jsx

```diff
@@ -1,8 +1,41 @@
 import React from 'react';
+import Header from '../components/header';
+import SearchResults from '../components/search-result';
 
-export default function Home(props) {
-  return (
-    <>
-    </>
-  );
+export default class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      restaurant: '',
+      location: '',
+      searchResults: null
+    };
+    this.handleSearch = this.handleSearch.bind(this);
+  }
+
+  handleSearch(searchParams) {
+    const req = {
+      method: 'GET',
+      headers: searchParams
```

Comment:
Instead of sending these in `headers`, you should send them in the query string of the url

```js
fetch('/api/businesses?restaurant=something&location=somewhere')
```

you can access these key-value pairs as `req.query` in your server code.

https://expressjs.com/en/4x/api.html#req.query